.github/workflows/integ_test.yml

```diff
@@ -55,46 +55,44 @@ jobs:
           project_path: ./AmplifyPlugins/Analytics/Tests/AnalyticsHostApp
           scheme: AWSPinpointAnalyticsPluginIntegrationTests
 
-  #  Disabling these tests because they do not currently work in CI. We will create a ticket to track
-  #  getting resolving this. In the meantime, they can be run locally.
-  #  
-  # push-notification-integration-test:
-  #   needs: prepare-for-test
-  #   runs-on: macos-12
-  #   environment: IntegrationTest
-  #   steps:
-  #     - uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b
-  #       with:
-  #         persist-credentials: false
-
-  #     - name: Make directory
-  #       run: mkdir -p ~/.aws-amplify/amplify-ios/testconfiguration/
-
-  #     - name: Copy integration test resouces
-  #       uses: ./.github/composite_actions/download_test_configuration
-  #       with:
-  #         resource_subfolder: push-notification
-  #         aws_role_to_assume: ${{ secrets.AWS_ROLE_TO_ASSUME }}
-  #         aws_region: ${{ secrets.AWS_REGION }}
-  #         aws_s3_bucket: ${{ secrets.AWS_S3_BUCKET_INTEG_V2 }}
-
-  #     - name: Set up node
-  #       uses: actions/setup-node@v3
-  #       with:
-  #         node-version: 16.x
-
-  #     - name: Run Local Server
-  #       run: |
-  #         cd ./AmplifyPlugins/Notifications/Push/Tests/PushNotificationHostApp/LocalServer
-  #         npm install
-  #         npm start &
-  #       shell: bash
-
-  #     - name: Run Integration test
-  #       uses: ./.github/composite_actions/run_xcodebuild_test
-  #       with:
-  #         project_path: ./AmplifyPlugins/Notifications/Push/Tests/PushNotificationHostApp
-  #         scheme: PushNotificationHostApp
+
+  push-notification-integration-test:
+    needs: prepare-for-test
+    runs-on: macos-12
+    environment: IntegrationTest
+    steps:
+      - uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b
+        with:
+          persist-credentials: false
+
+      - name: Make directory
+        run: mkdir -p ~/.aws-amplify/amplify-ios/testconfiguration/
+
+      - name: Copy integration test resouces
+        uses: ./.github/composite_actions/download_test_configuration
+        with:
+          resource_subfolder: push-notification
+          aws_role_to_assume: ${{ secrets.AWS_ROLE_TO_ASSUME }}
+          aws_region: ${{ secrets.AWS_REGION }}
+          aws_s3_bucket: ${{ secrets.AWS_S3_BUCKET_INTEG_V2 }}
+
+      - name: Set up node
+        uses: actions/setup-node@v3
```

Comment:
We should change this to reference the commit hash the corresponds to the v3 tag.

Reply:
updated the [v3 tag](https://github.com/actions/setup-node/releases/tag/v3)'s commit hash.